tests/cypress/integration/newfold-labs/wp-module-onboarding/steps/get-started-experience.cy.js

```diff
@@ -0,0 +1,44 @@
+// <reference types="Cypress" />
```

Comment:
Some more relevant tests can be
1. Testing that the drawer does not open on clicking drawer toggle button.
2. Iterating over all the radio buttons and making sure the correct button is checked on click, Ref: Tax Info PR
3. Ensuring that on initial load the `Continue Setup` button is disabled.

Reply:
1 and 3 is covered - Check if the Suppressed Drawer does not open on clicking Toggle Button, Enable Button based on the radio button selected
Have added a few more checks for radio buttons